### PR TITLE
feat(radio-button): testing

### PR DESCRIPTION
**Pull request by alexandereneroth**
Function
- [x] Value picked up by form.
- [x] Supports all native props.
- [x] Change event is fired with the right name when not disabled.
- [x] Change event is not fired when disabled.
- [x] All props/methods/events/states are tested and works.
- [x] It is possible to extract the values from input elements

Appearance
- [x] Default state looks like in Figma.
- [x] Hover state looks like in Figma.
- [x] Active state looks like in Figma.
- [x] Focus state looks like in Figma.
- [ ] Disabled state looks like in Figma.
❌ *Fail, checkbox is gray 700 should be gray 400*
- [x] Component specific states looks like in Fimga.

Consistency
- [ ] Props, methods, and events are consistent with other components

*Looks consistent to me*

### CODE REVIEW

File: src/components/RadioButton.tsx [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+        id: "iceCrsdfeam8234",
+      },
+    ],
+  },
+};
+
+const onSubmit = (e: FormEvent) => {
+  e.preventDefault();
+
+  console.log("onSubmit, event", e);
```
Comment (mJarsater) [inline src/components/RadioButton.tsx:33]: Maybe log the FormData here?

**Comment by mJarsater** [comment]
Since this one is mainly testing that it, and how it works within a form. Do you wanna add some examples, or build on the ones that are in the form page?

**Comment by alexandereneroth** [comment]
How do we verify consistent prop names?

Do we have a list of used prop names somewhere, if not maybe we could compile one.

**Comment by alexandereneroth** [comment]
> Since this one is mainly testing that it, and how it works within a form. Do you wanna add some examples, or build on the ones that are in the form page?

Hmm I did it now in the main page, would it be advantageous to have it in the form page?

**Review by mJarsater — APPROVED** [review]

**Pull request merged**